app/Http/Controllers/ProjectController.php

```diff
@@ -72,4 +66,14 @@ public function update(StoreCreateProject $request)
         }
         return response()->json(['message' => 'Project does not exist: '.$data['id']]);
     }
+    public function edit($id)
+    {
+        $project = Project::find($id);
+        return response()->json($project);
+    }
+
+    public function show($id)
```

Comment:
This method does not have a body. Please add a `// TODO` comment or remove this method if you don't plan on using it.